src/CodegenShapeFuture.php

```diff
@@ -0,0 +1,63 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2015-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HackCodegen;
+
+/**
+ * Generate code for a shape. Please don't use this class directly; instead use
+ * the function codegenShape_FUTURE.  E.g.:
```

Comment:
backticks around the funciton name will help automatic documentation generation in the near future

Reply:
i didn't see other classes doing this, but will add